### ISSUE TITLE
Audio slowing down video

### ISSUE THREAD

**Issue by jsalexjs** [Sunoo/homebridge-camera-ffmpeg#986]
Hi, 

I've just added audio to one of my cameras, everything is working as expected, however, the addition of audio makes the video wait around 5 seconds before begining the stream. Is there anyway to configure it so that video starts immediately and then audio starts streaming shortly after, as soon as its ready?

Any help would be appreciated thanks!

(yes I am using audio from a different camera that happens to have a mic)
{
                    "name": "Front Door Cam",
                    "manufacturer": "Hikvision",
                    "motion": true,
                    "doorbell": true,
                    "switches": true,
                    "motionTimeout": 60,
                    "videoConfig": {
                        "source": "-probesize 32 -analyzeduration 0 -re -rtsp_transport tcp -i rtsp://login:password@192.168.xx.xx/PSIA/Streaming/channels/1802/picture -f mulaw -ar 8000 -i http://login:password@192.168.xx.xx/bha-api/audio-receive.cgi",
                        "stillImageSource": "-probesize 32 -analyzeduration 0 -re -i rtsp://login:password@192.168.xx.xx/PSIA/Streaming/channels/1802/picture -f mulaw -i http://login:password@192.168.xx.xx/bha-api/audio-receive.cgi",
                        "maxStreams": 3,
                        "maxWidth": 1920,
                        "maxHeight": 1080,
                        "maxFPS": 20,
                        "maxBitrate": 9999,
                        "forceMax": false,
                        "vcodec": "libx264",
                        "packetSize": 564,
                        "audio": true,
                        "debug": true
                    }
                },

**Comment by Sunoo** [comment]
I can’t think of a good reason that would happen. Perhaps somehow related to the fact that they’re coming from two different sources? If you share your log, I might be able to take a guess as to what’s going on.

**Comment by jsalexjs** [comment]
Sure I imagine it may be something along those lines. Here's the log, thanks for your help :)

> [1/9/2021, 10:52:51 AM] [Camera FFmpeg] [Front Door Cam] Snapshot requested: 1280 x 720
[1/9/2021, 10:52:51 AM] [Camera FFmpeg] [Front Door Cam] Sending snapshot: 1280 x 720
[1/9/2021, 10:52:51 AM] [Camera FFmpeg] [Front Door Cam] Snapshot command: /usr/local/lib/node_modules/homebridge-camera-ffmpeg/node_modules/ffmpeg-for-homebridge/ffmpeg -probesize 32 -analyzeduration 0 -re -i rtsp://xxx:xxx@192.168.10.18/PSIA/Streaming/channels/1802/picture -f mulaw -i http://xxx:xxx@192.168.10.108/bha-api/audio-receive.cgi -frames:v 1 -filter:v scale='min(1280,iw)':'min(720,ih)':force_original_aspect_ratio=decrease,scale=trunc(iw/2)*2:trunc(ih/2)*2 -f image2 -
[1/9/2021, 10:52:51 AM] [Broadlink RM] Alex Bedroom TV getSwitchState: false
[1/9/2021, 10:52:52 AM] [Camera FFmpeg] [Backroom] Device appears to be inactive. Stopping stream.
[1/9/2021, 10:52:52 AM] [Camera FFmpeg] [Backroom] Stopped video stream.
[1/9/2021, 10:52:53 AM] [Broadlink RM] Alex Bedroom TV getSwitchState: false
[1/9/2021, 10:52:56 AM] [Broadlink RM] Alex Bedroom TV getSwitchState: false
[1/9/2021, 10:52:58 AM] [TuyaWebPlatform] [Alex Office] - Requesting device state
[1/9/2021, 10:52:58 AM] [TuyaWebPlatform] [Alex Office] - Creating new debounced promise
[1/9/2021, 10:52:58 AM] [TuyaWebPlatform] [Alex Office] - Triggering debouncedDeviceStateRequest
[1/9/2021, 10:52:59 AM] [TuyaWebPlatform] [Alex Office] - Unsetting debouncedDeviceStateRequestPromise
[1/9/2021, 10:52:59 AM] [TuyaWebPlatform] [Alex Office] - Resolving resolveDeviceStateRequest from remote
[1/9/2021, 10:52:59 AM] [TuyaWebPlatform] [Alex Office] - Set device state request cache
[1/9/2021, 10:52:59 AM] [TuyaWebPlatform] [Alex Office] Characteristic.CurrentTemperature - [GET] undefined
[1/9/2021, 10:53:00 AM] [Broadlink RM] Studio AC onTemperature (20.8)
[1/9/2021, 10:53:00 AM] [Broadlink RM] Studio AC onHumidity (28.22)
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] Video stream requested: 1280 x 720, 30 fps, 299 kbps
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] Starting video stream: 1920 x 1080, 20 fps, 600 kbps
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] Stream command: /usr/local/lib/node_modules/homebridge-camera-ffmpeg/node_modules/ffmpeg-for-homebridge/ffmpeg -probesize 32 -analyzeduration 0 -re -rtsp_transport tcp -i rtsp://xxx:xxx@192.168.10.18/PSIA/Streaming/channels/1802/picture -f mulaw -ar 8000 -i http://xxx:xxx@192.168.10.108/bha-api/audio-receive.cgi -an -sn -dn -codec:v libx264 -pix_fmt yuv420p -color_range mpeg -r 20 -f rawvideo -preset ultrafast -tune zerolatency -filter:v scale='min(1920,iw)':'min(1080,ih)':force_original_aspect_ratio=decrease,scale=trunc(iw/2)*2:trunc(ih/2)*2 -b:v 600k -payload_type 99 -ssrc 15677100 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params lA3RXwF/Z2/kq6RFKwLEA+pMCt2wGJtqJP8c+a12 srtp://192.168.10.98:65041?rtcpport=65041&pkt_size=564 -vn -sn -dn -codec:a libfdk_aac -profile:a aac_eld -flags +global_header -f null -ar 16k -b:a 24k -ac 1 -payload_type 110 -ssrc 9955322 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params 2gNkTHYg5/QHtlvoiDl+ANhIpHgu3TajaAzZM9hq srtp://192.168.10.98:63902?rtcpport=63902&pkt_size=188 -loglevel level+verbose
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info] ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   built with Apple clang version 11.0.3 (clang-1103.0.32.62)
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   configuration: --enable-videotoolbox --pkgconfigdir=/Users/runner/work/ffmpeg-for-homebridge/ffmpeg-for-homebridge/build/workspace/lib/pkgconfig --prefix=/Users/runner/work/ffmpeg-for-homebridge/ffmpeg-for-homebridge/build/workspace --pkg-config-flags=--static --extra-cflags=-I/Users/runner/work/ffmpeg-for-homebridge/ffmpeg-for-homebridge/build/workspace/include --extra-ldflags=-L/Users/runner/work/ffmpeg-for-homebridge/ffmpeg-for-homebridge/build/workspace/lib --extra-libs='-lpthread -lm' --enable-static --disable-debug --disable-shared --disable-ffplay --disable-doc --enable-openssl --enable-gpl --enable-version3 --enable-nonfree --enable-pthreads --enable-libvpx --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libx264 --enable-runtime-cpudetect --enable-libfdk-aac --enable-avfilter --enable-libopencore_amrwb --enable-libopencore_amrnb --enable-filters --enable-decoder=h264 --enable-network --enable-protocol=tcp --enable-libspeex --enable-demuxer=rtsp
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libavutil      56. 51.100 / 56. 51.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libavcodec     58. 91.100 / 58. 91.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libavformat    58. 45.100 / 58. 45.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libavdevice    58. 10.100 / 58. 10.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libavfilter     7. 85.100 /  7. 85.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libswscale      5.  7.100 /  5.  7.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libswresample   3.  7.100 /  3.  7.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   libpostproc    55.  7.100 / 55.  7.100
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a6073c0] [verbose] Starting connection attempt to 192.168.10.18 port 554
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a6073c0] [verbose] Successfully connected to 192.168.10.18 port 554
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [verbose] SDP:
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] v=0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] o=- 1109162014219182 0 IN IP4 0.0.0.0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] s=HIK Media Server V3.5.35
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] i=HIK Media Server Session Description : standard
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] e=NONE
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] c=IN IP4 0.0.0.0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] t=0 0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=control:*
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] b=AS:10250
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=range:npt=now-
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] m=video 0 RTP/AVP 96
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] i=Video Media
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=rtpmap:96 H264/90000
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=fmtp:96 profile-level-id=4D0014;packetization-mode=0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=control:trackID=video
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] b=AS:10240
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=Media_header:MEDIAINFO=494D4B48010100000400000100000000000000000000000000000000000000000000000000000000;
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] a=appversion:1.0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [verbose] setting jitter buffer size to 0
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936a801600] [error] non-existing PPS 0 referenced
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936a801600] [error] non-existing PPS 0 referenced
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936a801600] [error] decode_slice_header error
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936a801600] [error] no frame!
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [warning] Stream #0: not enough frames to estimate rate; consider increasing probesize
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [info] decoding for stream 0 failed
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [warning] Could not find codec parameters for stream 0 (Video: h264, 1 reference frame, none): unspecified size
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] Consider increasing the value for the 'analyzeduration' and 'probesize' options
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info] Input #0, rtsp, from 'rtsp://xxx:xxx%@192.168.10.18/PSIA/Streaming/channels/1802/picture':
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   Metadata:
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]     title           : HIK Media Server V3.5.35
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]     comment         : HIK Media Server Session Description : standard
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]   Duration: N/A, bitrate: N/A
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [info]     Stream #0:0: Video: h264, 1 reference frame, none, 90k tbr, 90k tbn, 180k tbc
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a4020c0] [verbose] Starting connection attempt to 192.168.10.108 port 80
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a4020c0] [verbose] Successfully connected to 192.168.10.108 port 80
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a4020c0] [verbose] Starting connection attempt to 192.168.10.108 port 80
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] [tcp @ 0x7f936a4020c0] [verbose] Successfully connected to 192.168.10.108 port 80
[1/9/2021, 10:53:01 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:01 AM] [Broadlink RM] Alex Bedroom TV getSwitchState: false
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [mulaw @ 0x7f936b800000] [warning] Estimating duration from bitrate, this may be inaccurate
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [warning] Guessed Channel Layout for Input Stream #1.0 : mono
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] Input #1, mulaw, from 'http://xxx:xxx@192.168.10.108/bha-api/audio-receive.cgi':
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]   Duration: 00:20:50.00, bitrate: 64 kb/s
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Stream #1:0: Audio: pcm_mulaw, 8000 Hz, mono, s16, 64 kb/s
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] Stream mapping:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]   Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]   Stream #1:0 -> #1:0 (pcm_mulaw (native) -> aac (libfdk_aac))
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] Press [q] to stop, [?] for help
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [NULL @ 0x7f936a801600] [error] non-existing PPS 0 referenced
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00e400] [error] non-existing PPS 0 referenced
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00e400] [error] decode_slice_header error
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00e400] [error] no frame!
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00ea00] [error] non-existing PPS 0 referenced
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00ea00] [error] decode_slice_header error
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00ea00] [error] no frame!
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [rtsp @ 0x7f936b001200] [warning] Thread message queue blocking; consider raising the thread_queue_size option (current value: 8)
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [h264 @ 0x7f936b00f000] [verbose] Reinit context to 640x480, pix_fmt: yuvj420p
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [error] Error while decoding stream #0:0: Invalid data found when processing input
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [error] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] Error while decoding stream #0:0: Invalid data found when processing input
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [Parsed_scale_0 @ 0x7f936a403cc0] [verbose] w:min(1920,iw) h:min(1080,ih) flags:'bicubic' interl:0
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [Parsed_scale_1 @ 0x7f936a404700] [verbose] w:trunc(iw/2)*2 h:trunc(ih/2)*2 flags:'bicubic' interl:0
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [graph 0 input from stream 0:0 @ 0x7f936a4052c0] [verbose] w:640 h:480 pixfmt:yuvj420p tb:1/90000 fr:90000/1 sar:4/3
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [Parsed_scale_0 @ 0x7f936a403cc0] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [verbose] w:640 h:480 fmt:yuvj420p sar:4/3 -> w:640 h:480 fmt:yuvj420p sar:4/3 flags:0x4
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [swscaler @ 0x1103c0000] [warning] deprecated pixel format used, make sure you did set range correctly
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [Parsed_scale_1 @ 0x7f936a404700] [verbose] w:640 h:480 fmt:yuvj420p sar:4/3 -> w:640 h:480 fmt:yuv420p sar:4/3 flags:0x4
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [libx264 @ 0x7f936b003000] [info] using SAR=4/3
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [libx264 @ 0x7f936b003000] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [libx264 @ 0x7f936b003000] [info] profile Constrained Baseline, level 3.0, 4:2:0, 8-bit
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] Output #0, rtp, to 'srtp://192.168.10.98:65041?rtcpport=65041&pkt_size=564':
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]   Metadata:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     title           : HIK Media Server V3.5.35
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     comment         : HIK Media Server Session Description : standard
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     encoder         : Lavf58.45.100
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Stream #0:0: Video: h264 (libx264), 1 reference frame, yuv420p(tv, left), 640x480 [SAR 4:3 DAR 16:9], q=-1--1, 600 kb/s, 20 fps, 90k tbn, 20 tbc
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Metadata:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]       encoder         : Lavc58.91.100 libx264
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Side data:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam]       
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] cpb: 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] bitrate max/min/avg: 0/0/600000 buffer size: 0 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] vbv_delay: N/A
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [mulaw @ 0x7f936b800000] [warning] Thread message queue blocking; consider raising the thread_queue_size option (current value: 8)
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [graph_1_in_1_0 @ 0x7f936a704400] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [verbose] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] tb:1/8000 samplefmt:s16 samplerate:8000 chlayout:0x4
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [format_out_1_0 @ 0x7f936a4083c0] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [verbose] auto-inserting filter 'auto_resampler_0' between the filter 'Parsed_anull_0' and the filter 'format_out_1_0'
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [auto_resampler_0 @ 0x7f936a634f80] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [verbose] ch:1 chl:mono fmt:s16 r:8000Hz -> ch:1 chl:mono fmt:s16 r:16000Hz
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] Output #1, rtp, to 'srtp://192.168.10.98:63902?rtcpport=63902&pkt_size=188':
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]   Metadata:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     title           : HIK Media Server V3.5.35
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     comment         : HIK Media Server Session Description : standard
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     encoder         : Lavf58.45.100
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Stream #1:0: Audio: aac (libfdk_aac) (ELD), 16000 Hz, mono, s16, delay 288, 24 kb/s
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]     Metadata:
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info]       encoder         : Lavc58.91.100 libfdk_aac
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] 
[1/9/2021, 10:53:03 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=    5 fps=0.0 q=25.0 size=      38kB time=00:00:00.33 bitrate= 929.4kbits/s speed=0.66x    
[1/9/2021, 10:53:04 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=   15 fps= 15 q=24.0 size=      83kB time=00:00:00.81 bitrate= 831.9kbits/s speed=0.809x    
[1/9/2021, 10:53:04 AM] [Camera FFmpeg] [Front Door Cam] [info] 
[1/9/2021, 10:53:04 AM] [Camera FFmpeg] [Front Door Cam] frame=   25 fps= 17 q=20.0 size=     101kB time=00:00:01.30 bitrate= 636.7kbits/s speed=0.861x    
[1/9/2021, 10:53:05 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=   35 fps= 17 q=17.0 size=     113kB time=00:00:01.80 bitrate= 513.9kbits/s speed=0.896x    
[1/9/2021, 10:53:05 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=   45 fps= 18 q=14.0 size=     130kB time=00:00:02.30 bitrate= 463.9kbits/s speed=0.911x    
[1/9/2021, 10:53:06 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=   56 fps= 18 q=15.0 size=     172kB time=00:00:02.89 bitrate= 486.7kbits/s speed=0.936x    
[1/9/2021, 10:53:06 AM] [Camera FFmpeg] [Front Door Cam] [info] frame=   67 fps= 18 q=13.0 size=     194kB time=00:00:03.40 bitrate= 467.5kbits/s speed=0.939x    
[1/9/2021, 10:53:06 AM] [Camera FFmpeg] [Front Door Cam] Stopped video stream.

**Comment by Sunoo** [comment]
I don’t see anything obviously wrong, you could try setting vcodec to copy and see if that helps at all, since it looks like your camera is spitting out h.264.

**Comment by jsalexjs** [comment]
Thanks for the suggestion, I've actually managed to fix the problem by using the same "-probesize 32 -analyzeduration 0" twice in my video source options. Once for the video segment and again for the sound segment. I've tried using "copy" for codec but some reason the end result actually ends up taking longer to load the videos stream. Counter intuitive I know but just how it seems to work on my system. Thanks for the help :)
